src/scripts/modules/header/index.js

```diff
@@ -501,7 +502,6 @@ class Header {
 
     const blacklist = [
       'climate.globalforestwatch.org',
-      'water.globalforestwatch.org',
```

Comment:
lets keep this line. I think Water need this for the links in the header when they render it.